src/js/edu-benefits/5490/config/form.js

```diff
@@ -241,7 +239,7 @@ const formConfig = {
                 }
               }),
               'ui:options': {
-                hideIf: (fieldData) => fieldData.relationship !== 'spouse'
+                hideIf: (pageData, allData) => _.get('applicantInformation.relationship', allData) !== 'spouse'
```

Comment:
Because the callback sends them in that order, so I can't just grab `allData` without first grabbing `pageData`. Hence my question above.